gis4wrf/plugin/plugin.py

```diff
@@ -89,6 +97,22 @@ def add_wps_binary_layer(self) -> None:
     def report_bug(self) -> None:
         webbrowser.open('https://github.com/GIS4WRF/gis4wrf/issues')
 
+    def check_version(self) -> None:
+        def on_succeeded(latest: str) -> None:
```

Comment:
I would re-order the two functions since get_latest_delayed is called first. Makes it easier to read.